ajax/libs/bootstrap-iconpicker/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bootstrap-iconpicker",
+  "version": "1.8.2",
+  "filename": "js/bootstrap-iconpicker.min.js",
+  "description": "A simple iconpicker for Bootstrap 3.x",
+  "homepage": "https://victor-valencia.github.io/bootstrap-iconpicker/",
+  "keywords": [
+    "bootstrap",
+    "iconpicker",
+    "icon-fonts",
+    "icons",
+    "css",
+    "javascript",
+    "jquery"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/victor-valencia/bootstrap-iconpicker.git",
+    "fileMap": [
+      {
+        "basePath": "bootstrap-iconpicker",
+        "files": [
+          "**/*"
```

Comment:
@victor-valencia Thank you for helping this PR make better.
Because we removed the version number in filenames. In auto-update config, we still want it to add the files of old versions and future versions as same with v1.8.2 file structure.

So, `"**/*"` need to change to `"**/+(bootstrap-iconpicker|iconset-all)*"`.

Many thanks.

Except for this part, I think this PR is fine.